apps/jupyterlab/app/app.sh

```diff
@@ -42,3 +42,15 @@ configure_jupyter() {
 if [ ! -f /opt/.jupyter_configured ]; then
     configure_jupyter
 fi
+
+echo "Starting Status Server: ..."
```

Comment:
we should avoid duplicating this. Same code is present on base/docker/scripts/start.sh

Reply:
Removed